docs/reference/2-ods-api/how-to-guides/how-to-create-and-manage-api-security-metadata-using-declarative-security-policies.mdx

```diff
@@ -411,24 +414,39 @@ apparently requires some manual manipulation to assemble into an XML document.
 
 The basic usage of the transformation script is as follows:
 
-```powershell
-.\transform.ps1 .\Security-Policies\MySecurityPolicy.xml .\Generated\MySecurityPolicy-Postgres.sql -DatabaseEngine PostgreSql
-.\transform.ps1 .\Security-Policies\MySecurityPolicy.xml .\Generated\MySecurityPolicy-SqlServer.sql -DatabaseEngine SqlServer
-```
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ```pwsh
+    .\transform.ps1 .\Security-Policies\MySecurityPolicy.xml `
+      .\Generated\MySecurityPolicy-Postgres.sql -DatabaseEngine PostgreSql
+    ```
+  </TabItem>
+  <TabItem value="MSSQL" label="SQL Server" default>
+    ```pwsh
+    .\transform.ps1 .\Security-Policies\MySecurityPolicy.xml `
+      .\Generated\MySecurityPolicy-SqlServer.sql -DatabaseEngine SqlServer
+    ```
+  </TabItem>
+</Tabs>
```

Comment:
This is only spot where I added a tabbed display. There are more examples that should be converted to use tabs in the future. I just didn't have time to look for them all. Here's how it renders:

![image](https://github.com/user-attachments/assets/3e1cfb84-e859-4b55-af98-b9684040a1a3)

Reply:
Sounds good. Adding to our notes. We may need to add additional guidance on when to use `<details>` over `<Tabs>`